### PR TITLE
Added TLS certificate in local MinIO

### PR DESCRIPTION
**Pull request by hanzlamateen**
## Summary

This PR replaces MinIO as storage provider for default development. You will have to update your `.env.local` file with the latest `.env.local.default` to make things work properly. 

### Local storage usage:

If you still want to use local storage then do following steps:

- Uncomment the values under `# Use following value for local file server` and comment out the values above it. `VITE_FILE_SERVER` & `STORAGE_PROVIDER`
- Instead of `npm run dev` use `npm run dev-fs`
- To reinitialize database use `npm run dev-reinit-fs` instead of `npm run dev-reinit`

### To be merged with this PR:
https://github.com/EtherealEngine/etherealengine-docs/pull/17
https://github.com/EtherealEngine/ethereal-engine-ops/pull/6
https://github.com/EtherealEngine/etherealengine-control-center/pull/149

## References

closes #_insert number here_

## Checklist
- [ ] If this PR is still a WIP, convert to a draft
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] [ensure all checks pass](https://github.com/etherealengine/etherealengine/wiki/Testing-&-Contributing)
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewer

## QA Steps

_List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos._

### CODE REVIEW

**Comment by hanzlamateen** [comment]
https://github.com/bitnami/containers/issues/36068

File: certs/minio/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# MinIO Certificates
+
+This certificate is generated using [certgen](https://github.com/minio/certgen) utility from MinIO.
+
+It was generated using following command:
+
+```bash
+certgen -host "127.0.0.1,localhost"
```
Comment (HexaField) [inline certs/minio/README.md:8]: If I use a different LAN IP to run everything from (such as 192.168.0.2), is this going to cause certification problems?

Comment (hanzlamateen) [reply certs/minio/README.md:8]: So can you please check by running this branch and see if minio is running fine in docker? And you can access it: https://etherealengine.github.io/etherealengine-docs/docs/installation/install_troubleshooting/#accessing-minio-s3-storage-provider-running-in-local-docker

If it doesnot then you will need to generate certificate for your ip.

**Comment by hanzlamateen** [comment]
@speigg seems like its working fine everywhere, just need to test it on macOS and mentioned in discord thread.

**Comment by mrhegemon** [comment]
What if we made our own docker container w/ nginx serving the TLS cert?

**Comment by hanzlamateen** [comment]
> What if we made our own docker container w/ nginx serving the TLS cert?

So, the TLS issue is resolved for MinIO. Its just docker engine that stops working on MacOS. Whereas its working totally fine on Windows and native Ubuntu

**Pull request merged**